Mathlib/AlgebraicGeometry/PrimeSpectrum/Basic.lean

```diff
@@ -933,7 +933,7 @@ variable (R) in
 Zero loci of prime ideals are closed irreducible sets in the Zariski topology and any closed
 irreducible set is a zero locus of some prime ideal.
 -/
-@[simps] def bijection :
+@[simps] protected def bijection :
```

Comment:
I think something like `orderEquivIrreducibleCloseds` would be a better name (there is [TopologicalSpace.Closeds](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Sets/Closeds.html#TopologicalSpace.Closeds) that you may use).

Reply:
Is irreducibleSetEquivPoints relevant here?

Reply:
Nice find. On first glance, it seems the order preserving is in the other direction. I will inspect the function more closely later.

Reply:
I have changed it to use `irreducibleSetEquivPoints`, thanks!